livekit-plugins/livekit-plugins-playai/livekit/plugins/playai/models.py

```diff
@@ -0,0 +1,15 @@
+from typing import Literal
+
+from pyht.client import Format  # type: ignore
+
+TTSModel = Literal["Play3.0-mini-ws", "PlayDialog-ws", "Play3.0-mini", "PlayDialog"]
+FORMAT = Literal["raw", "mp3", "wav", "ogg", "flac", "mulaw", "pcm"]
```

Comment:
let's include only the formats that the framework could decode. I'd recommend sticking with MP3 for now